charts/gateway/README.md

```diff
@@ -1157,6 +1159,46 @@ The full default is this
     com.l7tech.server.clusterStaleNodeCleanupTimeoutSeconds=86400
     # Additional properties go here
 ```
+### DISKLESS Configuraton
+DISKLESS_CONFIG is a flag to pass gateway sensitive data to be mounted to the container gateway file system or passed as environment variables.
+By Default, true, environment variables are used to configure Gateway.
+
+When DISKLESS_CONFIG is false , Gateway will be configured from node.properties
+
+#### node.properties
+Note: 
+- Database configuration should be same as mentioned in node.properties
+```
+node.cluster.pass=newpassword
+admin.user=admin
+admin.pass=newpassword
+node.db.config.main.host=myDBHost.com
+node.db.config.main.port=3306
+node.db.config.main.name=ssg
+node.db.config.main.user=gateway
+node.db.config.main.pass=newpassword
+```
+- For derby database, it is required to add ***node.db.type=derby*** to node.properties
+```
+node.cluster.pass=newpassword
+admin.user=admin
+admin.pass=newpassword
+node.db.type=derby
+node.db.config.main.user=gateway
+```
+
+- Mounting a pre-configured node.properties to container gateway
+
+```
+disklessConfig:
+  enabled: true
+  value:
+  # existingSecretName:
+```
+- Use set file to place a node.properties here
+ ```
+helm install my-ssg --set "disklessConfig.enabled=true" --set-file "disklessConfig.value=path/to/node.properties"  --set-file "license.value=license.value=path/to/license.xml" --set "license.accept=true"layer7/gateway  ./values.yaml
```

Comment:
- Duplicate license.value in the second --set-file
- Need space between "license.accept=true" and layer7/gateway

Reply:
Addressed in https://github.com/CAAPIM/apim-charts/pull/318/commits/9b9d8d060847d479d6993d9698df8e6afd3cda32